pkg/releasepayload/conditions/helpers.go

```diff
@@ -0,0 +1,86 @@
+package conditions
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+// ByReleasePayloadConditionType sorts a list of ReleasePayloadConditions by their Type
+type ByReleasePayloadConditionType []metav1.Condition
+
+func (in ByReleasePayloadConditionType) Less(i, j int) bool {
+	return in[i].Type < in[j].Type
+}
+
+func (in ByReleasePayloadConditionType) Len() int {
+	return len(in)
+}
+
+func (in ByReleasePayloadConditionType) Swap(i, j int) {
+	in[i], in[j] = in[j], in[i]
+}
+
+// The following logic is straight from: https://github.com/openshift/library-go/pull/1341
+// I'm just copying it here to prevent having to do the vendoring dance and we can pick
+// up the new version of library-go when the PR merges.
+
+func SetCondition(conditions *[]metav1.Condition, newCondition metav1.Condition) {
```

Comment:
followup: remove in favor of https://github.com/openshift/library-go/pull/1341 please